.github/workflows/publish_canary_docker.yml

```diff
@@ -0,0 +1,43 @@
+name: "devrelease:docker"
```

Comment:
Curious, why do we need this one?

Reply:
We can also remove the canary builds. They were useful for testing and Lushnikiv and me used them a lot so I think it brings value.